### ISSUE TITLE
Terminal hangs and never opens

### ISSUE THREAD

**Issue by kylebisley** [kontena/lens#7867]
**Describe the bug**
When launching a "Terminal Session" the terminal hangs displaying "Connecting . . ." indefinitely. I can connect to the cluster using terminals not launched through lens.

edit: Further information
I can connect directly to pod, and nodes, through lens however still cannot get an integrated terminal to load that is not attached to something in the cluster. 

Appears to have occurred since recent Lens upgrade.

**To Reproduce**
1. Connect to cluster.
2. New Tab -> Terminal Session

**Expected behavior**
A functional WSL Terminal using the kubectl context for selected cluster.

**Screenshots**
![image](https://github.com/lensapp/lens/assets/17894343/a6034224-bae7-4b19-b58b-25a9d63877ca)

**Environment (please complete the following information):**
Version:
Lens: 2023.5.310801-latest
Extension API: 6.5.2
Electron: 22.3.10
Chrome: 108.0.5359.215
Node: 16.17.1
Installation method... Do not recall. Likely simply followed the prompts at https://k8slens.dev/ for windows 10.

**Logs:**
```
> lens open --debug
time="2023-07-11T14:52:02-07:00" level=debug msg="load plugin configurations from lens API: get commands from api: client request: Get \"http://api/commands\": dial windows pipe: open api:80: The system cannot find the file specified."
time="2023-07-11T14:52:02-07:00" level=debug msg="failed to walk plugin dir: CreateFile .: The system cannot find the file specified."
time="2023-07-11T14:52:02-07:00" level=debug msg="found 0 file based plugin configurations"
time="2023-07-11T14:52:02-07:00" level=debug msg="sending telemetry: &{Action:open Metadata:map[arch:amd64 cols:120 commit:1a505e8 inputTTY:true isPre:false msDuration:20.4191ms msSetupDuration:16.2088ms os:windows outputTTY:true pluginCount:0 rows:30 success:true version:dev wsl:false]}"
time="2023-07-11T14:52:02-07:00" level=debug msg="failed to send telemetry payload: Post \"http://api/telemetry\": dial windows pipe: open api:80: The system cannot find the file specified."

```

**Kubeconfig:**
Uncomfortable posting my .kube/.config file as it is long and full of very sensative data. Obfuscating it would take a long time and everything works fine from the terminal.

**Additional context**
Worked fine yesterday.

**Comment by kylebisley** [comment]
Incase someone else finds this later I'll put in my solution. 
 
The issue was my default terminal stopped launching for some reason. I was using WSL and didn't make any intentional changes before it started failing. 

I opened PowerShell and ran 
`(get-command wsl).Path`
`>C:\Windows\system32\wsl.exe`
I opened Lens->File->Preferences->Terminal to ensure my `Terminal Shell Path` matched `wsl.exe ` and it was different! I corrected the discrepancy and low and behold all is well. 

TLDR:
My default terminal in lens became incorrectly set. 

_____
After the above fix [my new problem is well captured in another bug. ](https://github.com/lensapp/lens/issues/6021)

**Comment by yoholiao** [comment]
i fix the same problem by remove kubectl in `~/Library/Application Support/Lens/binaries/kubectl`（macos）。have a try。
